packages/tocco-ui/src/Select/StyledReactSelect.js

```diff
@@ -126,11 +123,11 @@ const reactSelectStyles = outerTheme => {
     singleValue: (base, state) => ({
       ...base,
       color: text[0],
-      margin: `0 ${spaceScale(-2)}`
+      margin: `0 ${scale.space(fakeProps, -2)}`
     }),
     valueContainer: (base, state) => ({
       ...base,
-      padding: `${spaceScale(-2)} ${spaceScale(-1)} 0 ${spaceScale(-2)}`
+      padding: `${scale.space(fakeProps, -2)} ${scale.space(fakeProps, -1)} 0 ${scale.space(fakeProps, -2)}`
```

Comment:
"fakeProps"? i dont get this name

Reply:
Method spaceScale expect props.theme as input. Inside reactSelectStyles props is inexistent, hence I define a constant (line 26) but name it a "fake". Still this constant does not differ from props. I could name it anyhow, it just must have contain a key named "theme". Should I name it "props"?

Reply:
i dont know. isn't spaceScale poorly designed if you have to enhance parameters like this? `fakeProps` definitely is a bad choice. It neither are props nor is something about that object fake since it does not fake a behavior.

Reply:
55288c9859a011e45b84b597d0cc29441ab67d24